bot/cogs/giveaway.py

```diff
@@ -0,0 +1,176 @@
+import logging
+from datetime import datetime, timedelta, timezone
+from typing import Optional
+
+import discord
+from discord import app_commands, Interaction
+from discord.ext import commands, tasks
+
+from bot.bot import WarnetBot
+from bot.config import BLACKLIST_GA_ROLE_ID, GUILD_ID
+
+logger = logging.getLogger(__name__)
+
+
+@commands.guild_only()
+class Giveaway(commands.GroupCog, group_name='warnet-ga'):
+    def __init__(self, bot: WarnetBot) -> None:
+        self.bot = bot
+        self.db_pool = bot.get_db_pool()
+
+    @commands.Cog.listener()
+    async def on_connect(self) -> None:
+        self._check_blacklist_ga.start()
+
+    @app_commands.command(name='blacklist', description='Blacklist a user from giveaway')
+    @app_commands.describe(
+        big='Yes if the giveaway is large than Rp50.000',
+        winners='Winner of the giveaway. Example: 1234567890,0987654321',
+        ghosts='Member who do not claim the giveaway. Example: 1234567890,0987654321',
```

Comment:
Should state that these two parameters contain discord ID

```suggestion
        winners='Discord IDs of the giveaway winner. Example: 1234567890,0987654321',
        ghosts='Member IDs who do not claim the giveaway. Example: 1234567890,0987654321',
```